archive.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+use Collection\GenresModel;
+use Collection\RecordsModel;
+
+require_once 'vendor/autoload.php';
+require_once 'src/displayAllGenresFunction.php';
+require_once 'src/dispayAllArchivedRecordsFunction.php';
+
+//connect and format db
+$db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
This code has been repeated a few times now, it's at the point where if you needed to change database details, you'd have to change it in a number of places. 

You should make a function that returns $db in a separate file, and then replace all other instances of PDO connetion code with that function.